addons/web/static/src/views/view_dialogs/select_create_dialog.xml

```diff
@@ -17,7 +17,12 @@
                 <t t-if="!props.noCreate">
                     <button class="btn btn-primary" data-hotkey="c" t-on-click="createEditRecord">Create</button>
                 </t>
-                <button class="btn o_form_button_cancel" t-att-class="(props.disableMultipleSelection &amp;&amp; props.noCreate) ? 'btn-primary' : 'btn-secondary'" data-hotkey="z" t-on-click="() => this.props.close()">Cancel</button>
+                <button class="btn o_form_button_cancel"
+                        t-att-class="(props.disableMultipleSelection &amp;&amp; props.noCreate) ? 'btn-primary' : 'btn-secondary'"
+                        data-hotkey="z" t-on-click="() => this.props.close()">
+                    <t t-if="props.onSelect or props.onCreateEdit">Cancel</t>
```

Comment:
Condition based on existence of props seems risky.

Can't we make it simple and *always* call it "close" ?

Reply:
Yes I though about it. Here the widget is used to view the record instead of its primary purpose that seems to create and select record. In other context, "cancel" seems more appropriate: cancel the selection or the creation.
I'll ask Luc and simplify as proposed if it is ok to.

Reply:
Yes it is approved, I have renamed the button "Close" for all cases. Thanks for the suggestion, it is indeed simpler.